kie-server-parent/kie-server-tests/kie-server-integ-tests-drools/src/test/java/org/kie/server/integrationtests/drools/pmml/PMMLApplyModelBaseTest.java

```diff
@@ -0,0 +1,25 @@
+package org.kie.server.integrationtests.drools.pmml;
+
+import org.junit.BeforeClass;
+import org.kie.api.KieServices;
+import org.kie.server.api.marshalling.MarshallingFormat;
+import org.kie.server.client.KieServicesConfiguration;
+import org.kie.server.client.RuleServicesClient;
+import org.kie.server.integrationtests.shared.basetests.RestJmsSharedBaseIntegrationTest;
+
+
+public abstract class PMMLApplyModelBaseTest extends RestJmsSharedBaseIntegrationTest {
+
+    protected RuleServicesClient rulesClient;
+
+    @BeforeClass
+    public static void setupFactory() throws Exception {
+        commandsFactory = KieServices.Factory.get().getCommands();
+    }
+
+    protected RuleServicesClient createRuleServicesClient() throws Exception {
+        KieServicesConfiguration cfg = createKieServicesRestConfiguration();
+        this.rulesClient = createDefaultClient(cfg, MarshallingFormat.JAXB).getServicesClient(RuleServicesClient.class);
```

Comment:
Is there a reason why you test just REST and JAXB? What about JMS and other marshalling formats?

Reply:
I added REST and JAXB tests as simple proofs of the ability to interact with the PMML when it is operating in a kie-server. I can add more tests, but my initial goal was to demonstrate that the API is available and able to be used.

Reply:
@sutaakar @lanceleverich 
Tests were updated to use all 3 types of marshallers. However, it seems that Xstream and JSON does not work.